saleor/graphql/schema.graphql

```diff
@@ -2254,6 +2295,10 @@ type Mutation {
   orderUpdateShipping(order: ID!, input: OrderUpdateShippingInput): OrderUpdateShipping
   orderVoid(id: ID!): OrderVoid
   orderBulkCancel(ids: [ID]!, restock: Boolean!): OrderBulkCancel
+  deleteMeta(id: ID!, key: String!): DeleteMeta
```

Comment:
Instead of using the word "meta" in these mutations I would call them like `deleteMetadata`, `deletePrivateMetadata` and so on.

Reply:
And the return type should `DeleteMetadata` etc.